src/schemas/json/partial-pyright.json

```diff
@@ -6,13 +6,7 @@
   "type": "object",
   "definitions": {
     "diagnostic": {
-      "anyOf": [
-        { "type": "boolean" },
-        {
-          "type": "string",
-          "enum": ["none", "information", "warning", "error"]
-        }
-      ]
```

Comment:
All except this fragment is about regex usage; this part drops unnecessary `anyOf`.